torchaudio/datasets/speechcommands.py

```diff
@@ -131,6 +132,27 @@ def __init__(
             walker = sorted(str(p) for p in Path(self._path).glob("*/*.wav"))
             self._walker = [w for w in walker if HASH_DIVIDER in w and EXCEPT_FOLDER not in w]
 
+    def get_metadata(self, n: int) -> Tuple[str, int, str, str, int]:
```

Comment:
```suggestion
    def get_metadata(self, n: int) -> Tuple[Union[str, Path], int, str, str, int]:
```

Reply:
It does not look like this function ever returns `Path`.